parse.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
```

Comment:
I took this file from `pycparser/examples/func_defs.py` and slightly modified it